enterprise/backend/src/metabase_enterprise/data_editing/api.clj

```diff
@@ -228,6 +228,36 @@
                                                :parameters]))]
       {:actions (vec (concat saved-actions table-actions))})))
 
+(mr/def ::api-action-id-saved
```

Comment:
Not sure how much overhead all this new indirection adds to our type-checking, but figure it could help thing make more sense on first encounter?

Or maybe the boilerplate and indirection hurts readability too much, and inline comments would be better?

@qnkhuat Let me know what you think.